.github/workflows/build.yml

```
@@ -10,16 +10,12 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - name: Dump GitHub context
-      env:
-        GITHUB_CONTEXT: ${{ toJson(github) }}
-      run: echo "$GITHUB_CONTEXT"
     - name: Set up Python 3.x (Latest)
-      uses: actions/setup-python@v4
+      uses: actions/setup-python@v5
       with:
         python-version: "3.x"
     - name: Checkout current repo
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
       with:
         submodules: true
```

This is a new setting that will suppress progress-bar-style output, which mostly just many lines to the logs. I made this change on other repos that I updated to `actions/checkout@v4`.
```suggestion
        submodules: true
        show-progress: false
```